specs/WinRT/WinRTAPIContracts.md

```diff
@@ -0,0 +1,80 @@
+# Windows App SDK API Contracts
+
+## Summary
+Customers trust that Microsoft products are sound. As Windows App SDK evolves and more developers
+use Windows App SDK in their own projects Windows App SDK needs a way, at build time and run time,
+to allow apps to light up API features and to gracefully degrade. This can be achieved by looking
+at the deltas between APIs.
+
+In order to allow apps to depend on API versions Windows App SDK will add contracts to all their
+WinRT APIs.
+
```

Comment:
NIT: Remove blank line

You only need 1 blank line between sections/paragraphs